test/command/9207-beamer-titlegraphicoptions.md

```diff
@@ -0,0 +1,223 @@
+```
+% pandoc -st beamer
+---
+titlegraphic: media/cdl23.pdf
```

Comment:
Let's leave out this test.  The problem is that it would need to be updated every time there's any change in the default LaTeX template, which gets to be a hassle.